config/routes.rb

```diff
@@ -6,6 +6,15 @@
 
   root "users#index" # Defines the root path route ("/")
 
+  # The following routes are for the API.
+  namespace :api do
+    namespace :v1 do
+      resources :posts, only: [:index]
+      resources :comments, only: [:index, :create]
+    end
+  end
```

Comment:
- [ ] Just as you did in thee normal render requesting a post to be assigned to a specific user, and making sure a specific user can be assed kindly edit your endpoint in that format for the APIs.

### Sample:
```ruby
Has API endpoint that returns all post in JSON format - GET : /api/users/:user_id/posts or /api/v1/users/:user_id/posts

Has API endpoint that returns all comments for a post - GET : /users/:user_id/posts/:id/comments

Has API endpoint that adds' a comment to a post - POST : /users/:user_id/posts/:id/comments
```

### Your view
![image](https://github.com/lily4178993/nell-blog/assets/50990925/4dee13fc-fbc3-495e-816a-1a7529b2e9a7)

Reply:
Hello @Shedrack-Sunday, thanks for the review. Could you please provide more specific feedback? I'm having difficulty understanding your point. The pull request description includes the specified endpoints, and you can create a JSON file based on the project description.